clang/cursor.go

```diff
@@ -45,20 +46,54 @@ func (c Cursor) PlatformAvailability(availabilitySize int) (always_deprecated bo
  */
 type CursorVisitor func(cursor, parent Cursor) (status ChildVisitResult)
 
+type functionsMapping struct {
```

Comment:
`s/functionsMapping/funcsMap/` ?
or `visitorRegistry` ?

(apologies for the bikeshedding)